### PR TITLE
This change updates the headlines in the Getting Started Guide.

### PR DESCRIPTION
**Pull request by petra-sargent**
*This change updates the headlines in the Getting Started Guid for CodeShip Pro

### CODE REVIEW

File: _pro/quickstart/getting-started-part-five.md [hunk 1]
```diff
@@ -91,20 +91,20 @@ Once the new build runs, we can check our log output and see our cache in action
 
 Caching is a really powerful way to speed your builds up. We also have a great article on optimizing your builds overall, as well as making sure your Dockerfile is designed with caching in mind. [You can read that here.](https://blog.codeship.com/speeding-up-your-docker-based-builds-with-codeship/)
 
-## Tests Per Branch
+### Running tags on the master branch
```
Comment (ethangj) [inline _pro/quickstart/getting-started-part-five.md]: I'd make this more generic, as it doesn't only apply to master - it can be any branch.

Comment (petra-sargent) [reply _pro/quickstart/getting-started-part-five.md]: How about: Running tags on a branch

**Comment by ethangj** [comment]
Works for me

File: _pro/quickstart/getting-started-part-five.md (hunk 1 above)
Comment (drewkitch) [reply _pro/quickstart/getting-started-part-five.md]: It's not very self evident what we mean when introduce a subject called "running tags on a branch". The fact that the attribute happens to be named `tag` is incidental (and unfortunate). Something along the lines of "whitelisting step commands by branch name" would be a _step_ in the right direction.

**Pull request merged**